PR-Template.txt

```diff
@@ -0,0 +1,7 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
+[ ] Each commit in the pull request should have a meaningful subject line and body.
+[ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.

```

Comment:
[CAMEL-XXX]: Fixes bug in camel-file component

Reply:
Ah so reword my PR request like [CAMEL-13704] Added PR template file?